doc/cascadia/profiles.schema.json

```diff
@@ -149,6 +149,29 @@
               "type": "boolean",
               "default": false,
               "description": "If true, the copied content will be copied as a single line (even if there are hard line breaks present in the text). If false, newlines persist from the selected text."
+            },
+            "copyFormatting": {
+              "default": null,
+              "description": "When set to `true`, the color and font formatting of selected text is also copied to your clipboard. When set to `false`, only plain text is copied to your clipboard. An array of specific formats can also be used. Setting this to null or not setting it inherits the global `copyFormatting` setting value.",
+              "oneOf": [
+                {
+                  "type": "boolean"
+                },
+                {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "enum": [
+                      "plain",
+                      "html",
+                      "rtf"
```

Comment:
is it possible to abstract this true/false/array=rtf,plain,html,all,none out into a shared type?